Fix:
error: implicit declaration of function 'utf8_char_width' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://rt.cpan.org/Public/Bug/Display.html?id=134342
--- strip_html.h.orig	2016-04-22 06:07:04.000000000 -0500
+++ strip_html.h	2022-01-06 00:37:21.000000000 -0600
@@ -33,6 +33,7 @@
 } Stripper;
 
 void _strip_html( Stripper * stripper, char * raw, char * clean, int is_utf8_p );
+int utf8_char_width(unsigned char * string);
 void _reset( Stripper * stripper );
 void clear_striptags( Stripper * stripper );
 void add_striptag( Stripper * stripper, char * tag );
